src/main/scanLogic/scanManager.ts

```diff
@@ -0,0 +1,56 @@
+import { ExtensionComponent } from '../extensionComponent';
+
+import { LogManager } from '../log/logManager';
+import { ConnectionManager } from '../connect/connectionManager';
+import { ConnectionUtils } from '../connect/connectionUtils';
+
+import { RootNode } from '../treeDataProviders/dependenciesTree/dependenciesRoot/rootTree';
+import { IGraphResponse, XrayScanProgress } from 'jfrog-client-js';
+import { GraphScanLogic } from './scanGraphLogic';
+
+/**
+ * Manage all the Xray scans
+ */
+export class ScanManager implements ExtensionComponent {
+    constructor(protected _connectionManager: ConnectionManager, protected _logManager: LogManager) {}
+
+    activate() {
+        return this;
+    }
+
+    /**
+     * Validate if the graph-scan is supported in the Xray version
+     */
+    private async validateGraphSupported(): Promise<boolean> {
+        let scanGraphSupported: boolean = await ConnectionUtils.testXrayVersionForScanGraph(
+            this._connectionManager.createJfrogClient(),
+            this._logManager
+        );
+        if (!scanGraphSupported) {
+            this._logManager.logError(new Error('scan with graph is not supported'), true);
+        }
```

Comment:
this will also show notification to the user that with this msg, the other one just logs on debug. i think we should show this error msg to the user to give feedback why the scan didn't produce results